docs/user/troubleshooting.md

```diff
@@ -0,0 +1,36 @@
+# Troubleshooting guide
+
+## Common troubleshooting techniques
+
+All Cluster Cloud Controller Manager Operator (CCCMO) components are deployed in `openshift-cloud-controller-manager` namespace. You can check their status by executing `oc get all -n openshift-cloud-controller-manager`.
+
+To see the operator logs use `oc logs -n openshift-cloud-controller-manager cluster-cloud-controller-manager-operator-<random suffix> -c cluster-cloud-controller-manager`.
+
+## Cluster Cloud Controller Manager Operator is degraded
+
+First, check the cluster operator status with `oc get clusteroperator cloud-controller-manager -oyaml` to find potential problems. Most likely the operator will have `Degraded` condition set to True.
```

Comment:
Nit, add a gap between the flag and argument so it's clear `yaml` is an argument
```suggestion
First, check the cluster operator status with `oc get clusteroperator cloud-controller-manager -o yaml` to find potential problems. Most likely the operator will have `Degraded` condition set to True.
```